### DISCUSSION TITLE
[Feature] Add netty related metrics

### DISCUSSION THREAD

**Discussion by swuferhong** [apache/fluss#93] [General]
### Search before asking

- [X] I searched in the [issues](https://github.com/alibaba/fluss/issues) and found nothing similar.

### Motivation

Fluss uses `Netty` as the underlying network communication component, but it does not expose Netty-related metrics, such as the amount of network memory allocation and the number of requests etc. This makes it less convenient for optimizing network performance.

### Willingness to contribute

- [ ] I'm willing to submit a PR!

**Comment by wuchong** [comment]
convert to issue #135
